### DISCUSSION TITLE
Can't click on StackPanel containing control with margin.

### DISCUSSION THREAD

**Discussion by MaxwellDAssistek** [AvaloniaUI/Avalonia#17229] [Q&A] [answered]
Hello,

I have a control (MaterialIcon in this case) that has a margin around it inside of a StackPanel. I'm expecting that the tapped event of the StackPanel triggers even when I click on the margin of the inner control, since the bounds of the StackPanel does contain it.

Here are a few screenshots showing the bounds of things using the developer tools.

Here is the margin of the inner control (the MaterialIcon): 
![image](https://github.com/user-attachments/assets/bc6cf00d-f313-444c-92a7-b33f217cdab3)

Here are the bounds of the StackPanel:
![image](https://github.com/user-attachments/assets/5ea15f30-2794-458a-8782-479ad2d55bc1)

Clicking where the red X is does not get recognized as a tap on the StackPanel.

I'm trying to make the click area around this button larger and I was hoping this would do it. How can I get the click there to be recognized?

**Comment by rabbitism** [comment, marked as answer]
set panel background to transparent
